### PR TITLE
mailmap: ztczll affiliation

### PR DESCRIPTION
**Pull request by ztczll**
Signed-off-by: zhanglei <243290414@qq.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
@ztczll you are replacing all the `\n` with `\r\n` in `.organizationmap`, could you adjust the setting of your editor, so it does not do the change?

**Comment by ztczll** [comment]
I have changed the Settings

File: .organizationmap [hunk 1]
```diff
@@ -718,7 +720,6 @@ UnitedStack <contact@unitedstack.com> Kun Huang <academicgareth@gmail.com>
 UnitedStack <contact@unitedstack.com> Li Tianqing <tianqing@unitedstack.com>
 UnitedStack <contact@unitedstack.com> Ning Yao <yaoning@unitedstack.com>
 UnitedStack <contact@unitedstack.com> wangsongbo <wangsongbo@unitedstack.com>
-UnitedStack <contact@unitedstack.com> Weibing Zhang <zhangweibing@unitedstack.com>
```
Comment (tchaikov) [inline .organizationmap]: see http://tracker.ceph.com/projects/ceph/wiki/Ceph_contributors_list_maintenance_guide#Mail-notification-to-the-author-about-an-update, maybe you can split this change into another commit.

if Webbing no longer affiliated to UnitedStack, maybe you can just update `.peoplemap` ? see http://tracker.ceph.com/projects/ceph/wiki/Ceph_contributors_list_maintenance_guide#Affiliation-change-over-time

File: .organizationmap [hunk 2]
```diff
@@ -508,6 +508,7 @@ The Linux Box <contact@linuxbox.com> Matt Benjamin <matt@linuxbox.com>
 The University of Arizona <contact@arizona.edu> James Ryan Cresawn <jrcresawn@gmail.com>
 Time Warner Cable Inc. <contact@twcable.com> Bryan Stillwell <bryan.stillwell@twcable.com>
 Trendy Tech <contact@trendytech.com.cn> shiqi <m13913886148@gmail.com>
+Trendy Tech <contact@trendytech.com.cn> zhanglei <243290414@qq.com>
```
Comment (tchaikov) [inline .organizationmap]: nit, maybe you could put in the way of `FirstName LastName` instead?

**Comment by ztczll** [comment]
I have separated the name,Thanks for review.
